### PR TITLE
bench: print statistics at the end

### PR DESCRIPTION
**Pull request by ldorau**
This change is [Reviewable](https://reviewable.io/reviews/pmem/vmemcache/100)

### CODE REVIEW

File: benchmarks/bench_simul.c [hunk 1]
```diff
@@ -232,6 +247,37 @@ static void *worker(void *arg)
...
 
+/*
+ * print_stats -- (internal) print all statistics
+ */
+static void
+print_stats(VMEMcache *cache)
+{
+	stat_t stat_vals[VMEMCACHE_STATS_NUM];
+
+#define GET_STAT(cache, stat_vals, i_stat)\
```
Comment (marcinslusarz) [inline benchmarks/bench_simul.c]: please don't use macros unless absolutely necessary

Comment (ldorau) [reply benchmarks/bench_simul.c]: Done

**Comment by ldorau** [comment]
@kilobyte please resolve your discussion

**Comment by ldorau** [comment]
It is ready

**Pull request merged**